src/Zicht/Bundle/HtmldevBundle/Resources/views/_base.html.twig

```diff
@@ -3,13 +3,14 @@
     <head>
         <meta charset="UTF-8">
         <meta http-equiv="x-ua-compatible" content="ie=edge">
-        <title>{% block head_title '' %}</title>
+        <title>{%- if app.environment != 'production' %}{{ assets_version() }} - {% endif %}{% block head_title '' %}</title>
 
         {% block head_copyright %}
             <!--
             Copyright (c) Zicht Online {{ 'now'|date('Y') }} All rights reserved
             Developed by: Zicht online
             Contact: info@zicht.nl
+            Version: {{ assets_version() }}
```

Comment:
Zou `Version:` geen `Assets version:` moeten heten? Kan mij zomaar voorstellen dat het opgevat kan worden als versie van de release (hash).

Reply:
Technisch gezien is het de 'Assets version', maar in de praktijk, en voor iedereen die dit in de HTML leest/opzoekt, is het de versie van de site die is live gezet.  Oorspronkelijk had ik dit ook de correct 'assets version' genoemd, maar ik kreeg er direct commentaar op door Sybolt.

Reply:
En... als je de deploy goed configureerd (zie aankomeende dev-meeting) dan staat er geen hash meer, maar een leesbare versie nummer.